releases/release-1.17/release-notes-draft.md

```diff
@@ -1,438 +1,100 @@
-## Action Required
```

Comment:
Did you specify the release branch as additional command line flag? (It looks like we're missing some notes here.)

Reply:
The exact command run is also the one used for the json file just used the markdown output instead
```
➜release-notes \
--branch=master \
--github-token=$GITHUB_TOKEN \
--repo-path /tmp/k8s \
--release-version 1.17.0-beta.1 \
--format markdown \
--output release-notes-1.17.0-beta.1.json \
--start-rev v1.17.0-beta.0 \
--end-rev v1.17.0-beta.1
```

Reply:
@cartyc should we be starting with alpha.0 so it encompasses all notes from the beginning? This differs from the json changes obviously.

Reply:
I think the `--branch` should be `release-1.17`

Reply:
@saschagrunert I can make the change but that is less consequential that which start and end tag I should use to make the notes as compared to patching the release-notes JSON. Also I will be putting in a PR to the documentation around this process because it is not very clear. As soon as I get confirmation from @cartyc on which start and end rev shuld be used for th markdown notes I will rerun and commit.  I will rebase as well to just include the final once I get the OK

Reply:
You ran the right thing for the Beta release if you can drop that json file into the assests, in the release notes site, https://github.com/kubernetes-sigs/release-notes that would be excellent :)

The start sha I have been using for the draft is `-start-sha d17cd2356993283316b10491a4beaad0931bbff3 `

Reply:
@saschagrunert @cartyc I am going to close this PR as I need to reset and commit on a branch of my fork, to avoid weird history issues with git. I have the process now for patches and the notes documented and will be PR'ing the draft notes based on the correct start SHA.
@cartyc I already did the patch to https://github.com/kubernetes-sigs/release-notes and its been merged alreadt so we shoule be good for release-notes now.